parquet-variant/tests/variant_interop.rs

```diff
@@ -63,9 +65,10 @@ fn get_primitive_cases() -> Vec<(&'static str, Variant<'static, 'static>)> {
         ("primitive_boolean_false", Variant::BooleanFalse),
         ("primitive_boolean_true", Variant::BooleanTrue),
         ("primitive_date", Variant::Date(NaiveDate::from_ymd_opt(2025, 4 , 16).unwrap())),
-        ("primitive_decimal4", Variant::Decimal4{integer: 1234, scale: 2}),
-        ("primitive_decimal8", Variant::Decimal8{integer: 1234567890, scale: 2}),
-        ("primitive_decimal16", Variant::Decimal16{integer: 1234567891234567890, scale: 2}),
+        ("primitive_decimal4", Variant::Decimal4(VariantDecimal4::try_new(1234, 2, 4).unwrap())),
```

Comment:
If you change the Variant::from impl above, this code would look like
```suggestion
        ("primitive_decimal4", Variant::from(VariantDecimal4::try_new(1234i32, 2u8, 10).unwrap())),
```

If you make the `TryFrom` impl, it would look like
```suggestion
        ("primitive_decimal4", Variant::try_from(1234, 2, 4).unwrap())),
```